app/src/main/java/com/github/factotum_sdp/factotum/ui/signup/SignUpFragment.kt

```diff
@@ -54,33 +54,40 @@ class SignUpFragment : Fragment() {
         val emailEditText = binding.email
```

Comment:
I think you should replace client id by username and just have it link directly to a contact username. I don't know what you think but since we place on linking contacts and profiles I don't think it is necessary for the user to write his full name in the sign up.